substra/sdk/models.py

```diff
@@ -25,9 +24,17 @@
 
 # The remote can return an URL or an empty string for paths
 UriPath = Union[FilePath, AnyUrl, str]
+CAMEL_TO_SNAKE_PATTERN = re.compile(r'(.)([A-Z][a-z]+)')
+CAMEL_TO_SNAKE_PATTERN_2 = re.compile(r'([a-z0-9])([A-Z])')
 
 
-class Status(enum.Enum):
+def _to_snake_case(camel_str):
+    name = CAMEL_TO_SNAKE_PATTERN.sub(r'\1_\2', camel_str)
+    name = CAMEL_TO_SNAKE_PATTERN_2.sub(r'\1_\2', name).lower()
+    return name.replace('_i_d', '_id')
```

Comment:
is this still necessary? I thought we are handling snake case everywhere.

Reply:
testtuple.traintuple_type is in camelCase as returned by the backend. From what I've seen, the type names are still in camelCase in the backend.

Reply:
oh come on! Well, it's the world we live in :) @Kelvin-M do you think this could be changed as part of the camel to snake case update ?